src/interface.jl

```diff
@@ -1,12 +1,23 @@
-# TODO: Improve docstring https://docs.julialang.org/en/v1/manual/documentation/
-"stage_states(mdp::MDP, stage::Int64)"
+"""
+Return number of state's stage
+"""
+function stage end
+
+"""
+Create Infinite Horizon MDP's states for given stage.
+"""
 function stage_states end
 
-# TODO: Improve docstring
-"stage_stateindex(mdp::MDP, s, stage::Int64)::Integer"
+"""
+Compute the index of the given state in Infinite Horizon for given stage state space.
+"""
 function stage_stateindex end
 
-# TODO: Docstring
+"""
+    HorizonLength(::Type{<:FHWrapper})
```

Comment:
```suggestion
    HorizonLength(::Type{<:Union{POMDP,MDP})
    HorizonLength(::Union{POMDP,MDP})
```
`HorizonLength(MyPOMDP) == FiniteHorizon()` means that every object of type MyPOMDP has a finite horizon. `HorizonLength(m) == FiniteHorizon()` means that the specific (PO)MDP object `m` has a finite horizon (this could depend on an object parameter. Does that makes sense to have both? I think it does.

Reply:
It definitely does.
I will commit these changes.
On the other hand, this is connected with #15. 
My opinion on this is that we should allow the user to implement his own FH (PO)MDP, but we should also force him either to use something to distinguish FH MDPs from IH MDPs or his definition should be of a specific type.
This way we will be able to differentiate between `HorizonLength(::Type{<:Union{POMDP, MDP})` and user-defined FH MDPs.
I think that this should be the topic of the next version (even if smaller), not this one.

Reply:
Going through code I realized that the user should also implement his own `HorizonLength(::Type{<:MyPOMDP})` method. That solves the issue and thus does not have to be dealt with.